invenio_rdm_records/services/components/access.py

```diff
@@ -19,13 +20,24 @@ class AccessComponent(ServiceComponent):
 
     def _populate_access_and_validate(self, identity, data, record, **kwargs):
         """Populate and validate the record's access field."""
+        errors = []
         if record is not None and "access" in data:
+            access = data.get("access")
+
+            # Explicit permission check for modifying record access.
+            # NOTE: This is very static/inflexible, but generalizing per-field
```

Comment:
I seem to recall something related to the `NOTE` keyword is actually used by some translation workflows so this line will end up in the translation files 😅 But Im not a 100% sure... maybe @mb-wali knows? (cannot find anything on google)

```suggestion
           # This is very static/inflexible, but generalizing per-field
```